WalletWasabi.Backend/wwwroot/about.html

```diff
@@ -0,0 +1,258 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="title" content="Wasabi Wallet - Bitcoin privacy wallet with built-in CoinJoin">
+    <meta name="description" content="Wasabi is an open-source, non-custodial, privacy-focused Bitcoin wallet for Desktop, that implements trustless CoinJoin over the Tor anonymity network.">
+    <meta http-equiv="onion-location" content="http://wasabiukrxmkdgve5kynjztuovbg43uxcbcxn6y2okcrsg7gb6jdmbad.onion" />
```

Comment:
This should not be here because it is different for every environment. It should be configured in nginx.

Reply:
Please do not mark problems as solved before addressing them or even comment about them.